examples/constant/constant.bal

```diff
@@ -0,0 +1,36 @@
+import ballerina/io;
+
+// Constants are defined using the `const` modifier.
+const string GET = "GET";
+
+// Constants can be defined without type using `var` keyword. Then the type will be inferred from the RHS.
+const POST = "POST";
+
+// These constants can be used to create type definitions. This type definition can only hold `GET` or `POST` values.
+type ACTION GET|POST;
+
+// Integer constant.
+const int I = 125;
+
+public function main() {
+
+    // This is identical to `ACTION get = "GET";`.
```

Comment:
Are we highlighting the quotation marks as an alternate syntax here? If so, better be clear on that point.